addons/payment_sips/controllers/main.py

```diff
@@ -36,15 +37,15 @@ def sips_ipn(self, **post):
             # meaningful information; log as a warning instead of a traceback
             _logger.warning('Sips: received empty notification; skip.')
         else:
-            self.sips_validate_data(**post)
+            self._sips_validate_data(**post)
         return ''
 
     @http.route('/payment/sips/dpn', type='http', auth="public", methods=['POST'], csrf=False)
```

Comment:
```suggestion
    @http.route(_return_url, type='http', auth="public", methods=['POST'], csrf=False)
```
same for `_notify_url`